README.md

```diff
@@ -153,6 +153,37 @@ The latest OMA Lightweight M2M IoT Agent documentation is also available on [Rea
 
 ---
 
+## Roadmap
+
+This product is a FIWARE Generic Enabler. If you would like to learn about the overall Roadmap of FIWARE, please check "Roadmap" on the [FIWARE Catalogue](https://www.fiware.org/developers/catalogue/).
+
+### Introduction
+
+This section elaborates on proposed new features or tasks which are expected to be added to the product in the foreseeable future.  There should be no assumption of a commitment to deliver these features on specific dates or in the order given. The development team will be doing their best to follow the proposed dates and priorities, but please bear in mind that plans to work on a given feature or task may be revised.  All information is provided as general guidelines only,  and this section may be revised to provide newer information at any time.
+
+### Short tem
+
+The following list of features are planned to be addressed in the short term, and incorporated in the next release of the product:
+- Docker image protects username and password using environment variables (#159).
+- Removal of warnings and deprecation warnings.
+- Usage example using [Eclipse Wakaama](https://github.com/eclipse/wakaama)
+- Creation of tutorial (#162)
+
+
+### Medium term
+
+The following list of features are planned to be addressed in the medium term, typically within the subsequent release(s) generated in the next 9 months after the next planned release:
+- Support to FIWARE complex data models.
```

Comment:
This is related with https://github.com/telefonicaid/iotagent-node-lib/issues/549 ? In that case, shouldn't be part of IOTA Node Lib roadmap instead of IOTAs roadmap?

Reply:
I think that something similar should be included also in the IOTA Node Lib, but for sure some kind of modification will be needed here since in the case of this IOTA, we need to consider the LWM2M objects mapping.

Reply:
Ok. Makes sense.

NTC